UPGRADE-v2.md

```diff
@@ -7,11 +7,15 @@ To upgrade to this version, you have to refactor your widgets "edit" and "new" t
 You don't have to redefine the "picker" div because it's been put outside the block in the parent template.
 
 ## UPGRADE 2.2.0
-You need to execute the following SQL command to upgrade your database
-UPDATE vic_link SET target = '_modal' WHERE target = 'ajax-modal'
+You need to execute the following SQL command to upgrade your database.
+```
```

Comment:
Please use this syntax to enable [syntax highlighting](https://help.github.com/articles/creating-and-highlighting-code-blocks/#syntax-highlighting):

    ```sql